src/index.js

```diff
@@ -0,0 +1,202 @@
+import './stylesheet.css';
```

Comment:
- [ ] Keep the application logic separated from DOM manipulation tasks.

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png)

Let's break our application in separate modules for modularity. I suggest that you add a `DOM` module which will take care of updating the results, showing the spinner, hiding the spinner, ..... It should't know any thing outside of its scope which is Dom manipulation.
We could also add another module for fetching the data from the weather API. You can add as many modules as you see fit, but make sure that you don't combine application logic with DOM manipulation tasks.